client/my-sites/marketing/buttons/buttons.jsx

```diff
@@ -95,6 +99,7 @@ class SharingButtons extends Component {
 			) {
 				this.props.successNotice( this.props.translate( 'Settings saved successfully!' ) );
 				this.props.markSaved();
+				// eslint-disable-next-line react/no-did-update-set-state
```

Comment:
Haven't made this change, but disabled the rule to be able to commit my changes.